source/MaterialXRuntime/Private/Commands/PvtSetMetadataCmd.cpp

```diff
@@ -0,0 +1,151 @@
+//
```

Comment:
Implementation of set metadata command. Note: this function can be used to either create a new metadata with a specified value or to simply change the value of an existing metadata.